test/models/test-enumeration-types.ttl

```diff
@@ -52,55 +52,46 @@
    samm:properties ( :partStatusAttributeValue :partStatusAttributeId [ samm:property :partStatusAttributeDescription; samm:notInPayload true ] ) .
 
 :StatusInProgress a :Status ;
-   :partStatusAttributeDescription "Minimalstatus PDM (obligatorisch): Neuanlage durch Entwicklung / Werk. Das Objekt wird durch die für die Erstellung verantwortliche Stelle bearbeitet."@de ;
    :partStatusAttributeDescription "In Progress"@en ;
```

Comment:
Create also a new test with an "Collection" and the corresponding Enumeration Instance for it with the "Parentheses" definition.

Reply:
I will fix the test for this also.